AnnoyancesFilter/sections/popups.txt

```diff
@@ -3583,9 +3583,9 @@ gamehunt.net#$#body { overflow: visible!important; }
 gamehunt.net#$#.modal { display: none!important; }
 gamehunt.net#$#.modal-backdrop { display: none!important; }
 thenation.com##.meerkat
-thenation.com#$#body { overflow: visible!important; }
-thenation.com#$#.tp-modal { display: none!important; }
-thenation.com#$#.tp-backdrop { display: none!important; }
+letemps.ch,thenation.com#$#body { overflow: visible!important; }
+letemps.ch,thenation.com#$#.tp-modal { display: none!important; }
+letemps.ch,thenation.com#$#.tp-backdrop { display: none!important; }
```

Comment:
letemps.ch#$#html > body.tp-modal-open { overflow: auto !important; }
letemps.ch#$#.tp-modal { display: none !important; }
letemps.ch#$#.tp-backdrop { display: none !important; }

They are needed, otherwise the rules in the pr won't work in the app.